core/src/main/java/feign/Client.java

```diff
@@ -65,7 +65,7 @@ public Default(SSLSocketFactory sslContextFactory, HostnameVerifier hostnameVeri
     @Override
     public Response execute(Request request, Options options) throws IOException {
       HttpURLConnection connection = convertAndSend(request, options);
-      return convertResponse(connection).toBuilder().request(request).build();
+      return convertResponse(connection, request).toBuilder().request(request).build();
```

Comment:
Actually, I can't.  The changes I made will require the request to exist before that.  The `toBuilder` creates a new `Builder` from an existing `Response`.  Since I made the `Request` mandatory, that call will throw an exception.

What I'll do is update it to not make the `toBuilder` call as I did elsewhere.  I must have missed this one during testing.